### ISSUE TITLE
What to do if an application wants to register the `user` RDF prefix?

### ISSUE THREAD

**Issue by wouterbeek** [ClioPatria/ClioPatria#23]
Upon loading ClioPatria the following RDF prefix is registered:

```prolog
:- rdf_register_ns(user, 'http://www.swi-prolog.org/cliopatria/user/').
```

The prefix name `user` may already be specified beforehand or afterwards by the... user.  In fact, `user` is -- not-so-commonly -- used to denote the IRI prefix `http://schemas.talis.com/2005/user/schema#` according to [prefix.cc](http://prefix.cc/user).

**Comment by wouterbeek** [comment]
I believe that if ClioPatria uses certain prefixes then the following two things should be ensured:
  1. If an application defines the same prefix differently then this should be undone, e.g., by using the `force/1` option.
  2. No application should be allowed to reset the ClioPatria-set prefix later.

Since (2) would limit the use of `rdf_register_prefix/[2,3]` it may be better to use less common prefix names, e.g., `cp-user` i.o. `user`.

**Comment by wouterbeek** [comment]
Another datapoint: ClioPatria sets `lib` to abbreviate `http://www.swi-prolog.org/rdf/library/` but an application may use the same prefix to abbreviate, e.g., `http://purl.org/library/`.

**Comment by JanWielemaker** [comment]
Yes.  It is probably best to use cp_* for ClioPatria's own prefixes.

**Comment by wouterbeek** [comment]
Looking at it now, it seems like namespace `user` is defined but not used anywhere.  Could this be the case?  Creating a user does not seem to add any triples. `?- rdf(S, P, O, G), rdf_global_id(user:Local, S).` also doesn't return anything.  Shall we just remove the prefix declaration?
